src/switch/Switch.tsx

```diff
@@ -0,0 +1,40 @@
+import "./_switch.scss";
+
+import React from "react";
+import classNames from "classnames";
+
+export interface SwitchProps {
+  onToggle: () => void;
+  isToggledOn: boolean;
```

Comment:
Can you use word `switch` instead of `toggle`?

Reply:
Yes, we can use it. But as typical usage, it similar to `Toggle` component behavior. Actually, I think `onToggle` more descriptive then. We can take the team ideas as well

Reply:
I also think `toggle` is OK here, but as an alternative, we can the name convention of checkbox directly:
```ts
isChecked
onChange
```
Maybe this would be a more predictable name for users

Reply:
Yeah, we can but `isChecked` looks better than `isToggledOn` for the checkbox element from the name consistency point.

Reply:
I think `onToggle` is fine because we are toggling the switch on and off whenever that function is executed. Maybe we can change `isToggledOn` as just `isOn` but they all look fine to me